exampleCourse/questions/demo/autograder/c/linkedListDelete/tests/test.py

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/python3
+
+import cgrader
+
+
+class TestGrader(cgrader.CGrader):
+    def tests(self):
+        self.test_compile_file(
+            "deletefirst.c",
+            "main",
+            add_c_file="/grade/tests/main.c",
+            flags="-I/grade/tests -fsanitize=address -static-libasan -g",
```

Comment:
Any reason you're manually passing the flags instead of using `enable_asan=True`?

Reply:
I updated the interface and forgot to update the example question 😬 ...

Reply:
Done in ac4c7a83c (also the call to `pl_setup_asan_hooks`).